Wikipedia/Code/WKWebView+LoadAssetsHtml.m

```diff
@@ -16,11 +18,12 @@ - (void)loadHTMLFromAssetsFile:(NSString*)fileName scrolledToFragment:(NSString*
     
     fragment = fragment ? fragment : @"top";
     NSURL* requestURL = [[WMFProxyServer sharedProxyServer] proxyURLForRelativeFilePath:fileName fragment:fragment];
-    NSURLRequest* request = [NSURLRequest requestWithURL:requestURL];
+    NSURLRequest* request = [NSURLRequest requestWithURL:requestURL cachePolicy:NSURLRequestReloadIgnoringCacheData timeoutInterval:WKWebViewLoadAssetsHTMLRequestTimeout];
```

Comment:
yeah- it should have had `NSURLRequestReloadIgnoringCacheData` even in the file case